src/pages/warehouse/[id].namespace.[nsid].vue

```diff
@@ -165,7 +165,13 @@
                     >
                   </td>
                 </template>
-
+                <template v-slot:item.actions="{ item }">
+                    <v-icon
+                      color="error"
+                      :disabled="!myAccess.includes('delete')"
```

Comment:
We got undrop on table and undrop all on namespace. If we want to offer the undrop button here and enable users who do not have undrop all on namespace to undrop, we'll have to send `getTableAccessById` for each table row which doesn't sound very efficient 

Dropping tabulars from this namespace view currently checks for delete on namespace access and does not additionally check via `getTableAccessById` although there is also a can_drop on table / view.

@v-kessler @c-thiel

Reply:
discussed with @c-thiel, we're gonna remove disabling from drop & undrop and just show an error if we fail to drop & undrop